tests/test_diffeq/test_wrappedscipyodesolution.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import pytest
+import pytest_cases
+from scipy.integrate._ivp import rk
+
+from probnum import diffeq
+from probnum.diffeq import wrappedscipysolver
+
+
+def case_logistic():
+    y0 = np.array([0.1])
+    ode = diffeq.logistic([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    testsolution = testsolver.solve(diffeq.AdaptiveSteps(0.1, atol=1e-2, rtol=1e-2))
+    scipysolution = testsolution.scipy_solution
+    return testsolution, scipysolution
+
+
+def case_lotkavolterra():
+    y0 = np.array([0.1, 0.1])
+    ode = diffeq.lotkavolterra([0.0, 1.0], y0)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    testsolution = testsolver.solve(diffeq.AdaptiveSteps(0.1, atol=1e-12, rtol=1e-12))
+    scipysolution = testsolution.scipy_solution
+    return testsolution, scipysolution
```

Comment:
thanks for migrating these! :) 

2 comments: (1) there is quite some code duplication here (the only difference between theses two cases are the first two lines, the other ones are copied and pasted -- I would be happy to see those factored out into a single function :)) (2) these ODE test cases exist in both the solution as well as the solver tests. Would be good to only define them once.

Reply:
I talked to @marvinpfoertner and @schmidtjonathan about this once again. I think _outsourcing_ the tests is better for this purpose. The solver cases can then also be used to test the noisy solvers.